docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -62,26 +55,19 @@ When you [Establish a Tax Provider connection](#establishing-a-connection), incl
   You must liaise with the merchant to provide them with customized URLs. BigCommerce merely consumes the URL values the merchant enters and uses the provided string value for Tax Provider API requests.
 </Callout>
 
-### Sandbox tax provider configuration
-
-Sandbox tax provider configurations are nested within your production tax provider configuration, as such tax providers requiring a sandbox tax provider configuration should also provide details for their production tax provider configuration.
+### Test tax provider (optional)
 
-Additionally, when providing the details for a sandbox tax provider configuration, it's unnecessary to specify the below properties because they are inherited from the primary production tax provider configuration (all other properties are still required):
+You can build a tax provider instance that supports test transactions. This test tax provider instance will sit alongside a primary production tax provider instance, servicing merchant-facing use cases or supporting the tax provider's own development processes.
```

Comment:
```suggestion
You can build a tax provider instance alongside a primary production tax provider instance. The test tax provider instance can support test transactions, service merchant-facing use cases, or support your own development processes. 
```

Reply:
Great, I've just adjusted this slightly to specify `a test tax provider instance` in the first sentence.

I've also modified the language in the following paragraph to better match this language ("test tax provider instance").